### DISCUSSION TITLE
When do you have plans to release log4j 2.17.0 ?

### DISCUSSION THREAD

**Discussion by cch4516** [strimzi/strimzi-kafka-operator#6068] [Q&A]
hi. thank you

When do you have plans to release log4j 2.17.0 ?
we are using strimzi kafka operator "0.24.0 and 0.26.1"  operator. 

Thank you always.

**Comment by ppatierno** [comment]
Hi, we are working to make a 0.27.0 release out in the coming days with the log4j 2.17.0 updated for all the components in the Strimzi ecosystem.
